ontology/spdx-ontology.owl.xml

```diff
@@ -307,7 +312,9 @@ Known issues:</rdfs:comment>
                 </owl:unionOf>
             </owl:Class>
         </rdfs:range>
-        <rdfs:comment xml:lang="en">The licensing that the preparer of this SPDX document has concluded, based on the evidence, actually applies to the package.</rdfs:comment>
+        <rdfs:comment xml:lang="en">The licensing that the preparer of this SPDX document has concluded, based on the evidence, actually applies to the package.
+
+If the Concluded License field is not present for an SPDX Item, it implies an equivalent meaning to NOASSERTION.</rdfs:comment>
```

Comment:
Comments refer to the RDF terms, so use `licenseConcluded` instead of `Concluded License`